.CI/build_all

```diff
@@ -25,4 +25,4 @@ git ls-tree --name-only master -- . | xargs -I {} sh -c "rm -rf {} && echo Remov
 popd > /dev/null
 
 # We just want to build all of the recipes.
-conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5"
+conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"
```

Comment:
I think we need the same constraint in `conda-smithy`.

ref: https://github.com/conda-forge/conda-smithy/blob/fe04e79d79ff6ae7d6e5b899283c7106b268759e/conda_smithy/configure_feedstock.py#L566